src/Home/CreateRoomButton.js

```diff
@@ -0,0 +1,26 @@
+import { supabase } from "../supabase_client";
+
+const CreateRoomButton = (props) => {
+  const { player, setPage, setRoomCode } = props;
+  const handleClick = async () => {
+    const roomKeys = "abcdefghijklmnopqrstuvwxyz";
+    let roomID = "";
+    for (let i = 0; i < 4; i++) {
+      roomID += roomKeys.charAt(Math.floor(Math.random() * roomKeys.length));
+      roomID = roomID.toUpperCase();
+    }
```

Comment:
Lets move this to its own function in a utils folder

Reply:
since that function is only used once, why should it be placed in a utils file?

Reply:
agree with anujan, doesnt need to be in a separate util if its only used once

Reply:
but should be in a separate function within this file